containers/runner/skip-testtypes

```diff
@@ -74,6 +72,14 @@ rhel10_skip_array=(
   rhel80086   # pykickstart removes md_ prefix from new MD devices
 )
 
+rhel10_skip_array=(
+  skip-on-rhel
+  skip-on-rhel-10
+)
+
+centos10_skip_array=(
```

Comment:
Is it intentionally empty? I would expect `skip-on-centos` and `skip-on-centos-10` here.

Reply:
Yes: [Run all tests on centos10 (TEMPORARY - for the next daily run).](https://github.com/rhinstaller/kickstart-tests/pull/1452/commits/6ddaf6ef0a875758faac1705929b570281d7808c)